src/app/core/entity-components/entity-select/entity-select/entity-select.component.ts

```diff
@@ -27,13 +28,33 @@ export type accessorFn<E extends Entity> = (E) => string;
   styleUrls: ["./entity-select.component.scss"],
 })
 export class EntitySelectComponent<E extends Entity> implements OnChanges {
+  /**
+   * The standard-type (e.g. 'Child', 'School', e.t.c.) to set.
+   * The standard-type has to be inside {@link ENTITY_MAP}
+   * @param type The type of entities that this will set. This will set the
+   * actual entity-type as well as the block-component
+   * @throws Error when `type` is not in the entity-map
+   */
+  @Input() set standardType(type: string) {
+    const entityType = ENTITY_MAP.get(type);
+    if (!entityType) {
+      throw new Error(`Entity-Type ${type} not in EntityMap`);
+    }
+    this.entityType = entityType;
+    if (DYNAMIC_COMPONENTS_MAP.has(type + "Block")) {
+      this.entityBlockComponent = type + "Block";
+    } else {
+      this.entityBlockComponent = undefined;
+    }
+  }
+  entityBlockComponent?: string;
   /**
    * The type of entity to load. This is required and will cause all
    * entities of the given type to be available in the selection
    * and auto-complete
    * @param type The type of the entity
    */
-  @Input() set entityType(type: EntityConstructor<E>) {
+  set entityType(type: EntityConstructor<E>) {
```

Comment:
Because its not a `@Input()` anymore , I think this can be just a normal function